login_required/middleware.py

```diff
@@ -46,11 +46,14 @@ def _login_required(self, request):
 
         return redirect_to_login(path, redirect_field_name=REDIRECT_FIELD_NAME)
 
-    def process_request(self, request):
+    def process_response(self, request, response):
         """
-        Use process_request instead of defining __call__ directly;
+        Use process_response instead of defining __call__ directly;
         Django's middleware layer will process_request in a coroutine in __acall__ if it detects an async context.
```

Comment:
shouldn't this also be replaced for `process_response`?

Reply:
Fair point, I've updated the docblock to mention both methods will be affected